### PR TITLE
KK-528 | Language tests

### PR DESCRIPTION
**Pull request by tuomas777**
Tests and a couple of bug fixes for languages app.

### CODE REVIEW

File: children/schema.py [hunk 1]
```diff
@@ -163,7 +163,7 @@ def validate_child_data(child_data):
 
 
 def set_child_languages_spoken_at_home(info, child, language_global_ids):
-    child.languages_spoken_at_home.all().delete()
+    child.languages_spoken_at_home.clear()
```
Comment (charn) [inline children/schema.py:166]: :sweat_smile: :+1:

Comment (tuomas777) [reply children/schema.py:166]: oops...

**Review by charn — APPROVED** [review]
LGTM! :+1: :shipit:

**Review by quyenlq — APPROVED** [review]
LGTM !!! 👌

**Pull request merged**